code/game/objects/structures/door_assembly.dm

```diff
@@ -686,4 +684,4 @@
 		overlays += get_airlock_overlay("fill_construction", icon)
 	else
 		overlays += get_airlock_overlay("[material]_construction", overlays_file)
-	overlays += get_airlock_overlay("panel_c[state+1]", overlays_file)
+	overlays += get_airlock_overlay("panel_c[state+1]", overlays_file)
```

Comment:
Blastdoor.dmi below: I changed the timing of opening/closing animations so they are the same length as firelock, airlock and doors (necessary since they now all use the same open() and close() proc with specific numbers in the sleep(X)).